### DISCUSSION TITLE
Should there be a way to prevent committing your own wuggestions in a PR?

### DISCUSSION THREAD

**Discussion by jayden-arrai** [community/community#31778] [Pull Requests]
Would it make sense to have a setting that can prevent this while viewing a pull request on github?

I searched and can't find anything related to this, so hopefully this isn't a duplicate discussion.

I made some suggestions to a PR and then on the Conversation tab of the PR I have the option to commit my own suggestions.  It seems a bit odd that I can commit my own suggestions on that screen.  Sure, nothing would prevent implementing my own suggestions and committing them to the branch and pushing them.

**Comment by airtower-luna** [comment]
I think you're giving your own answer here:

> Sure, nothing would prevent implementing my own suggestions and committing them to the branch and pushing them.

There's simply no point in preventing you from doing something via suggestions that you could also do by other means. If you wanted to commit directly you probably would've already done it. And if you later change your mind, well, why make it harder than necessary?
